dist-test/build.sh

```diff
@@ -14,6 +14,7 @@ fi
 # Run the build
 docker-compose \
     run \
+    --rm \
```

Comment:
Without this a new container is created on every run. Annoying when iterating locally.